types/enums/ApplicationTypes.ts

```diff
@@ -98,4 +98,8 @@ export const ApplicationTypes = {
     'Outline Planning Permission - Consent for the principle of a project specifying some details',
   treeWorksConsent:
     'Consent to carry out works to a tree in a Conservation Area or with a Tree Preservation Order',
+  'treeWorksConsent.conservationArea':
+    'Notification of proposed works to a tree in a Conservation Area',
+  'treeWorksConsent.tpo':
+    'Consent to carry out works to a tree with a Tree Preservation Order',
```

Comment:
Sorry for delay in reviewing these - but had a chance to catch up with planners & service designers on our side today and learned a couple useful things: 

- Tree services are currently being developed using the proposed application type `wtt` "Works to trees"
  - This prefix is both a bit more consistent with others (eg `ldc`, `pa`, `pp`) and removes the word "Consent" from the parent application type as that's actually incorrect as this change points out!
- Rather than use suffixes `.conservationArea` & `.tpo`, they've suggested `wtt.consent` & `wtt.notice` to better reflect the application language part of the description and more flexibly adapt to a future where other types of planning constraint designations may fall within either

So, what do you think about:
```
wtt: 'Works to trees',
'wtt.consent': 'Works to trees - Consent to carry out works to a tree with a Tree Preservation Order',
'wtt.notice': 'Works to trees - Notification of proposed works to a tree in a Conservation Area',
```

Realise this is annoyingly a breaking change now instead of an additive one, but hopefully quite harmless as we aren't actually exchanging this application type yet?

Reply:
Yes, won't be an issue on our side - do you want me to update the PR?

Reply:
Great - if you're happy to update here directly that sounds good to me! Thanks!